starknet_py/proxy/contract_abi_resolver.py

```diff
@@ -134,7 +134,7 @@ async def _get_class_by_address(self, address: Address) -> DeclaredContract:
                 class_hash=contract_class_hash
             )
         except ClientError as err:
-            if "is not deployed" in err.message:
+            if "is not deployed" in err.message or err.code == 28:
```

Comment:
I think we should extract this code to some constant and declare it at the top of the file (or in other place).